tests/publiccloud/google_cli.pm

```diff
@@ -30,7 +31,7 @@ sub run {
         assert_script_run "curl $url -o google-cloud-sdk.tar.gz";
         assert_script_run "tar xvf google-cloud-sdk.tar.gz";
         assert_script_run "google-cloud-sdk/install.sh --quiet --usage-reporting false --command-completion true";
-        assert_script_run "source ~/.bashrc";
+        assert_script_run "source ~/google-cloud-sdk/path.bash.inc";
```

Comment:
Why include the specific include instead of the generic bashrc? This invalidates the test, if the SDK is installed properly and setup in a way, that a new `bash` shell would work with it properly?

Reply:
No. I checked this and only command completion are put into `~/.bashrc` but the `gcloud` binary is still not visible under `$PATH`.